### PR TITLE
183 installing package in readme

### PR DESCRIPTION
**Pull request by Karim-Mane**
Changes in this PR address issue #183 raised by @joshwlambert .

### CODE REVIEW

**Review by joshwlambert — COMMENTED** [review]
I'm not sure this fully resolves #183 as the package is still installed using `pak::pak()` on L78 everytime the `README` is rendered. Instead I would recommend not evaluating this code chunk. To me the installation instructions are for package users on how to install the package rather than being required for the Quick start.

If you're thinking that the package needs to be installed in order to call `library(cleanepi)` then I don't think that is the case. Locally you should be able to render the `README` because the package will already be installed, and the `render_readme` workflow should be able to run thanks to `actions/checkout@v4` (I'm not 100% sure on this last point). 

Either way this is a personal choice so feel free to choose whatever works best for you and merge the changes whenever you're ready.

**Comment by Karim-Mane** [comment]
> I'm not sure this fully resolves #183 as the package is still installed using `pak::pak()` on L78 everytime the `README` is rendered. Instead I would recommend not evaluating this code chunk. To me the installation instructions are for package users on how to install the package rather than being required for the Quick start.

> thanks @joshwlambert - The `README.md` file does not show any sign of the package been installed as this is generated locally where `{pak}` already exists. Could you give an example of when this is been executed?

> If you're thinking that the package needs to be installed in order to call `library(cleanepi)` then I don't think that is the case. Locally you should be able to render the `README` because the package will already be installed, and the `render_readme` workflow should be able to run thanks to `actions/checkout@v4` (I'm not 100% sure on this last point).

> To my understanding, these installation instructions are only activated when rendering the `README.Rmd` on the remote server (on GitHub) to create the `README.md` file.

> Either way this is a personal choice so feel free to choose whatever works best for you and merge the changes whenever you're ready.

> We might need to discuss these during the next RSE meeting to harmonise on how they are handled internally, and add the consensus to the blueprint (if it is not already there).

**Comment by joshwlambert** [comment]
Sounds good, happy to discuss with the group.

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ The latest development version of **cleanepi** can be installed from
 [GitHub](https://epiverse-trace.github.io/cleanepi/).
 
 ``` r
-if (!("pak" %in% rownames(installed.packages()))) {
+if (!require("pak")) {
   install.packages("pak")
 }
```
Comment (Degoot-AM) [inline README.md:77]:
```suggestion
if (!require("pak")) install.packages("pak")
```

**Review by Degoot-AM — APPROVED** [review]

**Pull request merged**